CONTRIBUTING.md

```diff
@@ -236,9 +244,16 @@ echo '{"version": "4.4.3"}' > version.json
    cd percona-server-mongodb
```

Comment:
The changed part is ok, but there is also a difference in aws build command, pls change it as well